internal/util/kubernetes/dynamic_client.go

```diff
@@ -49,7 +46,7 @@ func ParseStatus(obj *GenericObject) (GenericStatus, error) {
 	return status, nil
 }
 
-func GetResource(
+func GetUnstructuredCR(
```

Comment:
Each method (`Get`, `List`, `Create`, and `Update`) now has a version which uses `Unstructured` type and a version which uses `GenericObject` type